waltid-services/waltid-wallet-api/src/main/kotlin/id/walt/webwallet/web/controllers/exchange/ExchangeController.kt

```diff
@@ -232,9 +231,8 @@ fun Application.exchange() = walletRoute {
             }
             response {
                 HttpStatusCode.OK to {
-                    body<CredentialOffer> {
-                        description = "Resolved credential offer"
-                    }
+                    description = "Resolved credential offer"
+                    body<CredentialOffer> {}
```

Comment:
just wondering..
why body here is called with an empty lambda, while above (body<String>) it's a regular function call
noticed these for Map as well